audio_video_recorder/README.md

```diff
@@ -0,0 +1,78 @@
+# audio_video_recorder
+
+ROS package for recording audio and video synchronously
+
+![[Full video on Google Drive](https://drive.google.com/file/d/1TWnRKbOdq6jPza82eNhhjn56lQXRxWjl/view?usp=sharing)
+](.media/pr2_sample.gif)
+
+[Full video on Google Drive](https://drive.google.com/file/d/1TWnRKbOdq6jPza82eNhhjn56lQXRxWjl/view?usp=sharing)
+
+## Sample
+
+You can record audio and video on your laptop.
+
+```bash
+roslaunch audio_video_recorder sample_audio_video_recorder.launch
+```
+
+## Parameters
+
+Node: `audio_video_recorder/audio_video_recorder`
+
+### Common parameters
+
+- `queue_size` (`Int`, default: `100`)
+
+  Queue size
+
+- `do_timestamp` (`Bool`, default: `true`)
+
+  Do timestamp or not (`false` is not working now.)
```

Comment:
what do you mean by `do timestamp` avi file have any property to set timepstamp?

Reply:
`do-timestamp` is not well documented in `gstreamer`, but it can cause gstreamer error in some programs (like `audio_common`).
Example is like this. https://github.com/ros-drivers/audio_common/pull/122
In `audio_common`, I figured out we can record correctly with `do-timestamp `false`, but in this program, do-timestamp `true` only works...
https://gstreamer.freedesktop.org/documentation/base/gstbasesrc.html?gi-language=c

Reply:
Anyway, I remove this parameter to avoid confusing users.